beacon-chain/core/epoch/precompute/reward_penalty.go

```diff
@@ -116,7 +116,7 @@ func attestationDelta(state *stateTrie.BeaconState, pBal *Balance, v *Validator)
 	finalityDelay := e - finalizedEpoch
 	if finalityDelay > params.BeaconConfig().MinEpochsToInactivityPenalty {
 		p += params.BeaconConfig().BaseRewardsPerEpoch * br
```

Comment:
Can you make `BaseRewardsPerEpoch` a local var? It's used like 3 or 4 times in this function.